src/vr.zig

```diff
@@ -2428,10 +2451,8 @@ pub const Replica = struct {
         // If client's last request id is <= this request id, then update client table entry.
         // Otherwise the client is already ahead of us, and we don't need to update the entry.
 
-        if (self.leader()) {
```

Comment:
We need to keep this branch, and yet still build the reply for followers without sending, because we're going to add a client table, to which followers will write the reply.